src/components/withEnvironment.tsx

```diff
@@ -54,9 +56,10 @@ export default function withEnvironment(WrappedComponent) {
         );
     });
 
-    WithEnvironment.displayName = `withEnvironment(${getComponentDisplayName(WrappedComponent)})`;
+    WithEnvironment.displayName = `withEnvironment(${getComponentDisplayName(WrappedComponent as ComponentType)})`;
```

Comment:
I would adjust the type of `getComponentDisplayName` to accept types with generics. WDYT?

Reply:
@kubabutkiewicz Yeah, @blazejkustra did it in [his PR](https://github.com/Expensify/App/pull/29209/files#diff-3401552ecd81836fbf3b40788fa80901ec80b064b37a99c75ead8cf1e38384deR4), so I'll update it when it's merged

Reply:
Okey, so LGTM 😄

Reply:
@VickyStash You can do the same change in this PR. Your PR is smaller, most likely it will be merged faster + it shouldn't produce conflicts if you copy same changes 😄